packages/server/src/definitions/automations.ts

```diff
@@ -20,9 +20,12 @@ export interface TriggerOutput {
 
 export interface AutomationContext {
   trigger: AutomationTriggerResultOutputs
-  steps: [AutomationTriggerResultOutputs, ...AutomationStepResultOutputs[]]
-  stepsById: Record<string, AutomationStepResultOutputs>
+  steps: Record<
```

Comment:
Previously we would track context in this shape and before each time we used it, we'd call a function called `prepareContext` that slightly transformed it. This ended up being quite expensive, so I made it such that the transformation is no longer needed.